vintage/__init__.py

```diff
@@ -52,14 +55,16 @@ def __call__(self, *args, **kwargs):
         warning = "{0} is deprecated".format(self._get_func_str())
         if self._message is not None:
             warning += ". {0}".format(self._message)
-        warn_deprecation(warning, frame_correction=+1)
+        warn_deprecation(warning, frame_correction=self._frame_correction)
```

Comment:
@vmalloc Oops ... Sorry...
I pushed again with a fix and unittest for validate it.
Please review again.